.circleci/config.yml

```diff
@@ -88,10 +88,10 @@ integration: &integration
           python -m pip install tox
     - run:
         name: install dependencies (node)
-        command: sudo -E bash tests/integration/ethers-cli/setup_node_v20.sh && sudo apt-get install -y nodejs
+        command: sudo -E bash tests/integration/js-integration-test-scripts/setup_node_v20.sh && sudo apt-get install -y nodejs
     - run:
-        name: Build ethers-cli
-        command: cd tests/integration/ethers-cli && sudo npm install && sudo npm install -g . && cd ../../../
+        name: Build js-integration-test-scripts
```

Comment:
This name seems redundant since it is located in the integration directory. I don't have context about the ethers tests or what that name was about. Do the tests encompass tests for any js client? Would be curious if there is a name that aligns with what the tests do?

Reply:
Ethers is to JS as web3py is to python - https://github.com/ethers-io/ethers.js/tree/e454afb2fa3612c273a09b154c36b35fafab17b1

Prior to this PR, the integration tests there only used a specific set of functions from ethers, so the name was appropriate. Since I've added tests that use both ethers and a metamask library, I figured a more general name was in order.

Reply:
Agree on the redundancy - I'll clean that up.

Reply:
Okey, shortened to `js-scripts`